packages/mendel-pipeline/src/daemon.js

```diff
@@ -163,6 +163,12 @@ module.exports = class MendelPipelineDaemon {
 
             this.cacheManager.sync(cache);
             this.pipelines[environment].watch();
+
+            // `config.support` is a special configuration
+            // that can dynamically add more entries to the pipeline.
+            if (envConf.support) {
+                this.watcher.subscribe([envConf.support]);
+            }
```

Comment:
Just wondering: Can we consider "support" a way to require something in all bundles? based on type? I don't fully understand the concept. Maybe add this to some readme? Do we have a readme for mendelrc anyway?